problemsets-mat223/modules/module1.tex

```diff
@@ -340,9 +340,9 @@
 Formalizing, for vectors $\vec u$, $\vec v$, $\vec w$, and scalars $\alpha$ and $\beta$, the
 following laws are always satisfied:
 \begin{align*}
-	(\vec u+\vec v)+\vec w&=\vec u+(\vec v+\vec w)\tag{Associativity}\\
-	\vec u+\vec v&=\vec v+\vec u\tag{Commutativity}\\
-	\alpha(\vec u+\vec v)&=\alpha\vec u+\alpha \vec v\tag{Distributivity}
+	(\vec u+\vec v)+\vec w&=\vec u+(\vec v+\vec w)\tag{Associativity\index{associativity}}\\
+	\vec u+\vec v&=\vec v+\vec u\tag{Commutativity\index{commutativity}}\\
```

Comment:
These aren't appropriate index references. Instead it should be `Laws of vector arithmetic` that should be in the index. Possibly with a `Vector operations!addition` and `Vector operations!scalar multiplication`